services/src/handlers/datasets.rs

```diff
@@ -1402,7 +1444,7 @@ mod tests {
 
         assert_eq!(
             meta_data,
-            MetaDataDefinition::OgrMetaData(StaticMetaData {
+            crate::datasets::storage::MetaDataDefinition::OgrMetaData(StaticMetaData {
```

Comment:
shouldn't this be the MetaDataDefinition from services/api/model ?

Reply:
I thought of `auto_detect_meta_data_definition` more of like an internal utility function which isn't directly connected to the API. Because of that I used internal types.